docs/contributing/support/support.rst

```diff
@@ -0,0 +1,44 @@
+Support
+=======
+
+Besides documentation we have a very friendly community which provides support for all Sylius users seeking help!
+
+At Sylius we have 3 main channels for communication and support.
+
+Slack
+-----
+
+Most of us use Slack for their online communication with co-workers. We know that it is not convenient to have another chat window open,
+therefore we've chosen Slack for the live communication with the community.
+
+We have setup following rooms so far:
+
+#general - for discussions about Sylius development itself,
+#docs - for discussions related to the documentation,
+#support - for asking questions and helping others.
+If you have any suggestions, please let us know in the comments section below this post.
+
+Slack requires inviting new members, but this can be done automatically, just go to `slackinvite.me/to/sylius-devs <https://slackinvite.me/to/sylius-devs>`_,
+enter your e-mail and you will get an e-mail with invitation, where you can setup your login and password.
```

Comment:
I'm conflicted about using `e-mail`, it's got a kind of 90's feel to it. Would suggest changing to the dashless version, but i guess it's a case of taste. On the other hand, using email in whatever form twice in a span of 7 words is a no no.
`set(space)up` btw